app/src/main/java/io/gnosis/safe/ui/safe/selection/SafeSelectionViewModel.kt

```diff
@@ -35,25 +35,28 @@ class SafeSelectionViewModel @Inject constructor(
 
     fun loadSafes() {
         safeLaunch {
-            val safes = safeRepository.getSafes()
             activeSafe = safeRepository.getActiveSafe()
+            val safes = safeRepository.getSafes().filter { it != activeSafe }.reversed()
 
             items.clear()
             items.add(AddSafeHeader)
+            activeSafe?.let {
+                items.add(it)
```

Comment:
Is this behaviour tested? That is, that the first is always the active safe if available.

Reply:
it is now